src/components/views/settings/ChangeAvatar.js

```diff
@@ -85,16 +82,15 @@ module.exports = React.createClass({
             newUrl = url;
             if (this.props.onAvatar) {
                 this.props.onAvatar(url);
```

Comment:
I think this needs to be `onUploadComplete` now (and references to it will also need updating)